docs/quickstart.md

```diff
@@ -308,23 +36,21 @@ This file should:
 (See the [Read Filtering Snakemake](readfilt_snakemake.md) page for details on
 these options.)
  
-```json
+Copy and paste the following:
+```
+cat > readfilt.json <<EOF
 {
     "files" : {
-        "SRR606249_1_reads.fq.gz" :           "files.osf.io/v1/resources/dm938/providers/osfstorage/59f0f9156c613b026430dbc7",
-        "SRR606249_2_reads.fq.gz" :           "files.osf.io/v1/resources/dm938/providers/osfstorage/59f0fc7fb83f69026076be47",
         "SRR606249_subset10_1_reads.fq.gz" :  "files.osf.io/v1/resources/dm938/providers/osfstorage/59f10134b83f69026377611b",
         "SRR606249_subset10_2_reads.fq.gz" :  "files.osf.io/v1/resources/dm938/providers/osfstorage/59f101f26c613b026330e53a",
-        "SRR606249_subset25_1_reads.fq.gz" :  "files.osf.io/v1/resources/dm938/providers/osfstorage/59f1039a594d900263120c38",
-        "SRR606249_subset25_2_reads.fq.gz" :  "files.osf.io/v1/resources/dm938/providers/osfstorage/59f104ed594d90026411f486"
     },
 
     "workflows" : {
         "read_filtering_pretrim_workflow" : {
-            "sample"    : ["SRR606249_subset25","SRR606249_subset10"],
+            "sample"    : ["SRR606249_subset10"],
         },
         "read_filtering_posttrim_workflow" : {
-            "sample"    : ["SRR606249_subset25","SRR606249_subset10"],
+            "sample"    : ["SRR606249_subset10"],
             "qual"   : ["2","30"]
         },
     },
```

Comment:
The parameters that follow are all the default values, so we can remove `direction_labels`, filename suffixes, or any other parameters the user would not need to worry about. The reason they are here is because I copied over the entire dictionary of default parameter values.